spark/ingestion/src/main/scala/feast/ingestion/stores/deadletters/DeadLetterMetrics.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2018-2020 The Feast Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package feast.ingestion.stores.deadletters
+
+import org.apache.spark.SparkEnv
+import org.apache.spark.metrics.source.DeadLetterSinkMetricSource
+
+object DeadLetterMetrics {
+  def writeMetrics() = {
```

Comment:
this method doesn't really write any metrics, they are being written asynchronously, so this name is missleading